src/index.js

```diff
@@ -0,0 +1,144 @@
+let previousSurgery;
+let diabetes;
+let hypertension;
+
+form.addEventListener("submit", function(event) {
+    event.preventDefault(); //stops page from reloading
+
+    // get BMI value from form
+    const weight = document.getElementById("weight").value;
+    const height = document.getElementById("height").value;
+    bmiValue = weight / (height / 100 * height / 100);
+
+    // create container div for result and associated div
+    const containerDiv = document.createElement("div");
+    containerDiv.className = "bmi-result";
+
+    // show appropriate div based on BMI value and add to container div
+    if (bmiValue < 18.5) {
+        underweightDiv.style.display = "block";
+        containerDiv.appendChild(document.createElement("p").appendChild(document.createTextNode(`Your BMI is ${bmiValue.toFixed(1)}.`)));
+        containerDiv.appendChild(underweightDiv);
+    } else if (bmiValue >= 18.5 && bmiValue < 25) {
+        normalDiv.style.display = "block";
+        containerDiv.appendChild(document.createElement("p").appendChild(document.createTextNode(`Your BMI is ${bmiValue.toFixed(1)}.`)));
+        containerDiv.appendChild(normalDiv);
+    } else if (bmiValue >= 25 && bmiValue < 30) {
+        overweightDiv.style.display = "block";
+        containerDiv.appendChild(document.createElement("p").appendChild(document.createTextNode(`Your BMI is ${bmiValue.toFixed(1)}.`)));
+        containerDiv.appendChild(overweightDiv);
+    } else if (bmiValue >= 30 && bmiValue < 40) {
+        obeseDiv.style.display = "block";
+        containerDiv.appendChild(document.createElement("p").appendChild(document.createTextNode(`Your BMI is ${bmiValue.toFixed(1)}.`)));
+        containerDiv.appendChild(obeseDiv);
+    } else {
+        morbidlyObeseDiv.style.display = "block";
+        containerDiv.appendChild(document.createElement("p").appendChild(document.createTextNode(`Your BMI is ${bmiValue.toFixed(1)}.`)));
+        containerDiv.appendChild(morbidlyObeseDiv);
+    }
+
+    // append container div to result and show "show-questions-btn"
+    const result = document.getElementById("result");
+    result.appendChild(containerDiv);
+    if (bmiValue > 25) {
+        showQuestionsBtn.style.display = "block";
+    }
+    resetBtn.style.display = "block";
+    form.style.display = "none";
+});
+
+    // show intervention questions when "show-questions-btn" is clicked
+    showQuestionsBtn.addEventListener("click", function() {
+    interventionQuestions.style.display = "flex";
+    showQuestionsBtn.style.display = "none";
+});
+
+// get user answers to intervention questions and recommend appropriate surgery
+const recommendBtn = document.getElementById("recommend-btn");
+
+const interventionsWrap = document.querySelector('.items-wrap');
+const itemGastricSleeve = document.querySelector('.item:nth-child(1)');
+const itemGastricBypass = document.querySelector('.item:nth-child(2)');
+const itemGastricBand = document.querySelector('.item:nth-child(3)');
```

Comment:
Ai putea să te folosești de proprietatea [`children`](https://developer.mozilla.org/en-US/docs/Web/API/Element/children).

```suggestion
const [
    itemGastricSleeve,
    itemGastricBypass,
    itemGastricBand
] = interventionsWrap.children;
```